test/e2e/components/vl-template.js

```diff
@@ -1,23 +1,21 @@
 const { VlElement } = require('vl-ui-core').Test;
 const { VlHeader } = require('vl-ui-header').Test;
 const { VlFooter } = require('vl-ui-footer').Test;
-const { By } = require('selenium-webdriver');
+const { By } = require('vl-ui-core').Test.Setup;
 
-class VlTemplate extends VlElement {  
+class VlTemplate extends VlElement {
     async getHeader() {
-        const header = await this.findElement(By.css('#header'));
-        return new VlHeader(this.driver, header);
+        return new VlHeader(this.driver);
```

Comment:
VlTemplate mag niet gekoppeld zijn aan VlHeader of VlFooter

Reply:
Zou inderdaad naar de demo page moeten